documentation/docs/steps/checkChangeInDevelopment.md

```diff
@@ -4,7 +4,22 @@
 
 ## Prerequisites
 
-* No prerequisites
+* For type SOLMAN
```

Comment:
@radsoulbeard : we mention the types SOLMAN and CTS here. RFC is misisng. This is that old so that is should work always, right ??? Or do we restrictions also wrt RFC?